codalib/bagatom.py

```diff
@@ -42,7 +42,8 @@ def wrapAtom(xml, id, title, author=None, updated=None, author_uri=None, alt=Non
             entryTag,
             ATOM + "link",
             rel='alternate',
-            href=alt)
+            href=alt,
+            type="text/html")
```

Comment:
Yeah. I worried about that, too. I did it under the assumption the `alt` kwarg is only used for this specific purpose. But I can add another kwarg so it can be explicitly set or default to `text/html`. Or I could omit it, though I feel like the usefulness of the `link` element is somewhat diminished, and the examples in the spec and related documents tend to include the `type` attribute.

Which do you think makes more sense?